### PR TITLE
Add org.kde.kpat

### PR DESCRIPTION
**Pull request by tsdgeos**
Based on the nightly one we have in KDE CI

### CODE REVIEW

File: org.kde.kpat.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                        }
+                    ]
+                },
+                {
+                    "name": "pbr",
+                    "buildsystem": "simple",
+                    "build-commands": [
+                        "pip3 install --prefix=/app --no-deps ."
+                    ],
+                    "sources": [ { "type": "git", "url": "https://github.com/openstack/pbr.git", "branch": "5.4.4" } ]
```
Comment (bilelmoussaoui) [inline org.kde.kpat.json]: Use a commit here too or probably a tarball if there are any.

File: org.kde.kpat.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
+{
+    "id": "org.kde.kpat",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "kpat",
+    "finish-args": ["--share=ipc", "--socket=x11", "--socket=wayland" ],
```
Comment (bilelmoussaoui) [inline org.kde.kpat.json]: Nitpick: put a finish-arg per line.

**Comment by bilelmoussaoui** [comment]
bot, build org.kde.kpat

File: org.kde.kpat.json (hunk 2 above)
Comment (tsdgeos) [reply org.kde.kpat.json]: It's a useless change and increases the diff with the nightly .json file we have in KDE CI, so i'm not doing this

**Comment by tsdgeos** [comment]
meh the stupid content rating thing

File: org.kde.kpat.json (hunk 2 above)
Comment (bilelmoussaoui) [reply org.kde.kpat.json]: or just fix the manifest upstream? it's just harder to read this way.

File: org.kde.kpat.json [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                            "url": "https://pypi.io/packages/source/p/pysol_cards/pysol_cards-0.8.7.tar.gz",
+                            "sha256": "b723c94ee5ccaa86ac40208cdd0b10e30f403f0c739731e5484e53a7932eecf8"
+                        }
+                    ]
+                },
+                {
+                    "name": "perl-modules",
+                    "buildsystem": "simple",
+                    "build-commands": [
+                        "echo 'this is a bit sad'",
```
Comment (bilelmoussaoui) [inline org.kde.kpat.json]:
```suggestion
```
Not needed.

Comment (tsdgeos) [reply org.kde.kpat.json]: Makes my life happier

File: org.kde.kpat.json (hunk 2 above)
Comment (tsdgeos) [reply org.kde.kpat.json]: Or we could leave it like this and save time discussing over what is more or less readable :)

File: org.kde.kpat.json (hunk 3 above)
Comment (tsdgeos) [reply org.kde.kpat.json]: I mean you understand i obviously know the echo is not needed, right?

**Comment by tsdgeos** [comment]
Anyhow closing this until we get the stupid oars content stuff into the next tarball

**Pull request closed without merging**